docs/resources/model_serving.md

```diff
@@ -68,6 +68,17 @@ The following arguments are supported:
 * `served_model_name` - (Required) The name of the served model this route configures traffic for. This needs to match the name of a `served_models` block
 * `traffic_percentage` - (Required) The percentage of endpoint traffic to send to this route. It must be an integer between 0 and 100 inclusive.
 
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Serving endpoint identifier (equal to `name` argument).
```

Comment:
```suggestion
* `id` - Equal to the `name` argument and used to identify the serving endpoint.
```
Just making it more clear that this attribute is effectively a dupe and to make it easier to distinguish from `serving_endpoint_id`.